aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -189,12 +189,9 @@ Tensor sparse_coo_tensor(const Tensor& indices, const Tensor& values_, const Ten
       sparse_dim, dense_dim, computed_sizes, indices, values, values.options().layout(kSparse));
 }
 
-// NB: Got rid of the sizes == NULL case
-Tensor sparse_coo_tensor(const Tensor& indices, const Tensor& values_, ArrayRef<int64_t> size, const TensorOptions& options) {
+bool _validate_sparse_coo_tensor_args(const Tensor& indices, const Tensor& values_, ArrayRef<int64_t> size) {
```

Comment:
It's a bit misleading that this returns a bool, but actually will never return false (and instead will just raise an error message). I think raising error messages is right, but then maybe the return type should just be `void` in that case.

Reply:
Neither void, nor None work as a result type in native_function.yaml. Or at least I couldn't find a way to make them work.

Reply:
lol, it's because it's spelled (). So `blah_blah(...) -> ()`